inc/server/class-stripe-server.php

```diff
@@ -105,6 +105,7 @@ public function register_routes() {
 	 * @access  public
 	 */
 	public function get_products( \WP_REST_Request $request ) {
+		$this->stripe = new Stripe_API();
```

Comment:
This can be a locally scoped variable now that we are using different instances of the class.